src/services/routePathLinkService.ts

```diff
@@ -1,15 +1,16 @@
 import { ApolloQueryResult } from 'apollo-client';
 import apolloClient from '~/util/ApolloClient';
 import IRoutePathLink from '~/models/IRoutePathLink';
-import IExternalLink from '~/models/externals/IExternalLink';
 import notificationStore from '~/stores/notificationStore';
 import NotificationType from '~/enums/notificationType';
 import RoutePathLinkFactory from '~/factories/routePathLinkFactory';
+import IExternalRoutePathLink from '~/models/externals/IExternalRoutePathLink';
+import IExternalLink from '~/models/externals/IExternalLink';
 import GraphqlQueries from './graphqlQueries';
 
 // TODO: create two services, RoutePathLinkService and LinkService?
 export default class RoutePathLinkService {
-    public static async fetchLinksWithLinkStartNodeId(nodeId: string)
+    public static async fetchAndCreateRoutePathLinksWithStartNodeId(nodeId: string)
```

Comment:
IMO we could just stick to arrow functions everywhere.. How do you feel on this?

Reply:
This could surely be discussed, but all other functions in all other service layers are done in this way. So, maybe we don't start refactoring all services in this task